.github/workflows/codspeed.yml

```diff
@@ -26,6 +26,22 @@ jobs:
           pdm venv create --with-pip --force $PYTHON
           pdm install -G testing -G testing-extra -G email
 
+      # override pydantic-core from git - to report benchmarks
+      # FIXME DO NOT MERGE THIS
+
+      - uses: actions/checkout@v4
+        with:
+          repository: pydantic/pydantic-core
+          ref: boxy/validator_serializer_reuse
+          path: pydantic-core
```

Comment:
Cool, we should probably do this against `pydantic-core` main sometimes.

Reply:
definitely seems like the kind of thing that could make sense to do before a release